Unable to build on Mac OS X 10.11

When attempting to build on Mac OS X 10.11 following the instructions in the README, the following error is produced:

```
bundler.cc:14:10: fatal error: 'wire/wire.hpp' file not found
#include <wire/wire.hpp>
         ^
1 error generated.
```

--- a/README.md
+++ b/README.md
@@ -34,7 +34,12 @@ Options:
         (*): Specify as many encoders as desired. Bundler will evaluate and choose the best compressor for each file.
 ```
 
-### Build
+### Getting Started
+
+```
+git clone https://github.com/r-lyeh/bundler --recursive
+```
+
 ```
 echo windows && cl bundler.cc -I deps /Ox /Oy /MT /DNDEBUG /DBUNDLE_USE_OMP_TIMER=1 /EHsc /link setargv.obj 
 echo windows && mt.exe -manifest bundler.exe.manifest -outputresource:bundler.exe;1

